### ISSUE TITLE
When there are multiple modules project referring third party module an error gets thrown

### ISSUE THREAD

**Issue by miyurud** [ballerina-platform/ballerina-lang#19268]
**Description:**
I get this error when trying to build the project attached with this github issue using "ballerina build -a". The console output I get is as follows,

```
Compiling source
	wso2/utils:0.1.0
	wso2/holder:0.1.0
	wso2/issuer:0.1.0
	wso2/chat:0.1.0

Creating balos
	target/balo/utils-2019r3-java8-0.1.0.balo
	target/balo/holder-2019r3-any-0.1.0.balo
	target/balo/issuer-2019r3-any-0.1.0.balo
	target/balo/chat-2019r3-any-0.1.0.balo
error: unable to copy native jar: /home/miyurud/git/proj1/blockchain-iam-group-chat/target/tmp/utils.jar
```

**Steps to reproduce:**
Download and extract the attached zip file ([blockchain-iam-group-chat.zip](https://github.com/ballerina-platform/ballerina-lang/files/3675543/blockchain-iam-group-chat.zip)). Change directory to "blockchain-iam-group-chat" folder and issue "ballerina build -a". You may also have to change the path of the _platform.libraries_ entry of the _Ballerina.toml_ file to match with your environment.

**Affected Versions:**
Ballerina 1.0.0

**OS, DB, other environment details and versions:**
Ubuntu 16.04
[blockchain-iam-group-chat.zip](https://github.com/ballerina-platform/ballerina-lang/files/3675543/blockchain-iam-group-chat.zip)

**Related Issues (optional):**
https://github.com/ballerina-platform/ballerina-lang/issues/19266

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by Kishanthan** [comment]
Possible duplicate of https://github.com/ballerina-platform/ballerina-lang/issues/18998 which is already fixed in 1.0.1 release.

**Comment by warunalakshitha** [comment]
I checked with 1.0.2 and I couldn't reproduce the issue. hence closing.
